scripts/diag.sh

```diff
@@ -0,0 +1,186 @@
+#!/bin/sh
+#
+# Copyright (C) 2023 Foundries.io
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Suggested setting with vim:
+# set ai ts=4 sw=4 et
+#
+# Authors:
+#    Tim Anderson <tim.anderson@foundries.io>
+#    Vanessa Maegima <vanessa.maegima@foundries.io>
+#    Caio Pereira <caio.pereira@foundries.io>
+
+VERSION=${VERSION-1.0}
+
+echo "*** diag tool version ***"
+echo "Version: $VERSION"
+echo
+
+echo "*** os-release ***"
+cat /etc/os-release
+echo
+
+test -f /var/sota/current-target && echo -e "*** current_target ***\n$(cat /var/sota/current-target)"
```

Comment:
I think this is the first target script in this repo. 

I would prefer to have a straightforward way to look at the file and see if it's supposed to be run on the target (instead of the host). Maybe, create a subfolder "target", and maybe change the script filename. Maybe when testing `/var/sota/current-target` print an error saying it's supposed to be on target?

Reply:
moved it to device-scripts